### PR TITLE
Add disable decimals property to textfield

### PR DESCRIPTION
**Pull request by constanzauanini**

### CODE REVIEW

File: src/components/Textfield/Textfield.tsx [hunk 1]
```diff
@@ -109,6 +116,10 @@ export default function TextField(props: Props): JSX.Element {
     }
   }
 
+  const preventDecimals = (event: React.KeyboardEvent<HTMLInputElement>) => {
```
Comment (charlie-tf) [inline src/components/Textfield/Textfield.tsx:119]: Is this called from anywhere?

Comment (constanzauanini) [reply src/components/Textfield/Textfield.tsx:119]: Oh, no. Will delete it, thanks

**Review by karthikbtf — APPROVED** [review]
i understand why we are adding this..

File: src/components/Textfield/Textfield.tsx [hunk 2]
```diff
@@ -33,6 +33,7 @@ export interface Props {
   tooltipTitle?: TooltipProps['title'];
   min?: number;
   max?: number;
+  disableDecimals?: boolean;
```
[the remark this answers is not in the document]
Comment (constanzauanini) [reply src/components/Textfield/Textfield.tsx:36]: Thanks, I like that option. I will refactor it!

**Review by karthikbtf — APPROVED** [review]

**Pull request merged**